### PR TITLE
OSX build for clustalo with OpenMP

### PR DESCRIPTION
**Pull request by colinbrislawn**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by colinbrislawn** [comment]
Do you know if I have to run `make` twice, once for each `configure`, or if I can configure everything then run make one time?

**Comment by souravsingh** [comment]
Configuring once should be fine

**Comment by colinbrislawn** [comment]
I want to end up with two executables. One for single threaded or systems without OpenMP, and a second on compiled with OpenMP. FastTree2 uses this setup and it works well.

Do you think we should do that or only have an OpenMP version of ClustalO on bioconda?

**Comment by colinbrislawn** [comment]
Also, I'm getting this error:
```
#!/bin/bash -eo pipefail
.circleci/setup.sh
From ssh://github.com/bioconda/bioconda-recipes
 * [new branch]          master     -> upstream/master
The CI configuration is out of date.
Please merge in bioconda:master.
Exited with code 1
```

Any idea? This looks like a simple error but I'm not sure how to solve it.

**Comment by mbargull** [comment]
@colinbrislawn, my bad concerning the error. Can you merge or rebase on master again? I made some additional changes to the CI configuration around the time you merged the first time. Sorry for the bother.

**Comment by colinbrislawn** [comment]
Sure thing! 

I know this is a basic git question, but what's the best way to go about doing this, without just opening a new PR? How do make my branch point to a newer master?

**Comment by colinbrislawn** [comment]
@brwnj Can you give me some feedback on this PR? I'm not sure if this is the best way to run the build to enable OpenMP. Let me know if you have any suggestions!

**Comment by colinbrislawn** [comment]
Turns out this is only needed on OSX. It already works on Linux because
>ClustalO will automatically support multi-threading if your compiler supports OpenMP.

 🙄 I'll try to get a simpler build that also works on OS!

File: recipes/clustalo/meta.yaml [hunk 1]
```diff
@@ -17,8 +17,6 @@ requirements:
     - llvm # [osx]
     - libgcc # [linux]
     - argtable2
-    
-    
 
 build:
   number: 0
```
Comment (rvalieris) [inline recipes/clustalo/meta.yaml]: you need to increase the build number, otherwise no build is triggered.

**Comment by colinbrislawn** [comment]
On OSX I'm getting
`clang: error: unsupported option '-fopenmp'` 

Would it help to switch to another version of gcc?
```
You could use a non-Apple gcc installed via MacPorts, adding

CC=/opt/local/bin/gcc-mp-4.5

to the configure call (you will have to change the exact string to match
your gcc version).
```

**Comment by colinbrislawn** [comment]
Could @rvalieris or @souravsingh make sure that the all gcc build is set up correctly? I think it's good to merge, but I wanted to make sure!

**Comment by rvalieris** [comment]
yes, it should be alright, thanks !

**Pull request merged**